src/INFO.sh

```diff
@@ -37,7 +37,7 @@ if [ "$DSM_VERSION" = "6" ]; then
   os_max_ver="7.0-40000"
 else
   os_min_ver="7.0-40000"
-  os_max_ver=""
+  os_max_ver="8.0-00000"
```

Comment:
I believe we were told to leave this empty on dsm 7?

Reply:
removed and verified.